pages/Workflow.md

```diff
@@ -10,14 +10,48 @@ A Developer wishing to design and deploy his/her own data services with Hydra, [
 
 In general, an Hydra network is made up by the interaction of an instance of Hydra agent with one or more instances of hydrus. An Hydra agent is an HTTP client with capabilities of translating REST queries from-to a graph query language, so to allow efficient navigation of the graph's internal representation (see [this paragraph](/Design#hydrus-as-a-full-stack-module) in Design).
 
-Follow on to [Example](Example.md) for a more in depth explanation.
+Follow [Example](Example.md) shown below for a more in depth explanation.
+
+# Flock Demo
+
+Assume a small simulation which consists of a Web API featuring a flock of drones. You can find demo [here](http://flockdemo.hydraecosystem.org). It basically consists of 3 major things.
+GUI: It’s the first thing you see when you open the demo (and also the only thing which you can see). You can use it to interact with the server. We also refer to it as hydra python agent.
+Controller: It’s a central hydra server which has the powers of starting, stopping drones and also giving instructions to them.
+Flock of drones: A group of drones which aim to detect fire or abnormal heat spots in a given geographical area, each drone can be active or not active at a given moment. They receive instructions from Controller and update controller every 15 sec.
+
+Assuming that you have already visited the demo. You might have come across various sections and should be curious what each one of them really means. So let’s study them one by one. We’ll go from left to right, would first study all the sections at left side to the map and then would study sections right to it. So here we go!
+
+First up there is input field which you can use to give instructions to individual drone, for eg you can change the speed of the drone by writing Set Drone 1 speed 10. As you might have already guessed. This message is first sent to controller via GUI and then controller takes some action depending on input supplied to it. 
+
+Next up is the Refresh Button, which you can use to reload active drones at any moment.
+
+Next up is the HTTP-API Logs Section. When some instructions are sent to drone they are sent via HTTP requests which can be either PUT or POST. This section logs all that activity. Now we are done with our left side and lets hop over to the right side and see what we can find over there.
+
+In this side, we've got two log sections similar to controller log we talked about moments ago. Remember we logged type of request and target over there, and in the first section we log purpose of that request. Now you may be wondering what do mean by purpose where does it come from? So For your information it’s the same thing which you typed in the input field we discussed earlier and sent to controller. As we already know that each drone updates controller every 15 sec. So in the second/last section we log updates sent from drone to controller.
```

Comment:
Not only the commands that user types by everything gets logged. Please try to read the source code for a better understanding of the demo and all other tools

Reply:
Yes I think it would be the best option